client/src/components/pages/Home.js

```diff
@@ -23,5 +24,17 @@ function Home() {
         </Container >
     )
 }
+=======
+import app from "./base";
+
+function Home() {
+    return (
+        <>
+        <h1>Home</h1>
+        <button onClick={() => app.auth().signOut()}>Sign Out</button>
+        </>
+    );
```

Comment:
I think this is just stashed here? We won't need a signOut button on the homepage because user will only be directed here if they are not logged in.